pkgs/applications/virtualization/colima/default.nix

```diff
@@ -1,54 +1,65 @@
 { lib
-, buildGo118Module
+, buildGoModule
 , fetchFromGitHub
 , installShellFiles
 , lima
 , makeWrapper
+, qemu
+, testers
+, colima
 }:
 
-buildGo118Module rec {
+buildGoModule rec {
   pname = "colima";
-  version = "0.4.2";
+  version = "0.4.4";
 
   src = fetchFromGitHub {
     owner = "abiosoft";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-66nKH5jxTzLB9bg2lH1E8Cc0GZ6C/N/+yPYhCVEKOBY=";
-
+    sha256 = "bSBaSS+rVkFqTSdyegdE/F0X5u7yvF/nHslAO3xgD6I=";
     # We need the git revision
     leaveDotGit = true;
     postFetch = ''
-      git -C $out rev-parse HEAD > $out/.git-revision
+      git -C $out rev-parse --short HEAD > $out/.git-revision
       rm -rf $out/.git
     '';
   };
 
   nativeBuildInputs = [ installShellFiles makeWrapper ];
 
-  vendorSha256 = "sha256-91Ex3RPWxOHyZcR3Bo+bRdDAFw2mEGiC/uNKjdX2kuw=";
+  vendorSha256 = "jDzDwK7qA9lKP8CfkKzfooPDrHuHI4OpiLXmX9vOpOg=";
 
-  doCheck = false;
+  CGO_ENABLED = 1;
 
   preConfigure = ''
-    ldflags="-X github.com/abiosoft/colima/config.appVersion=${version}
-              -X github.com/abiosoft/colima/config.revision=$(cat .git-revision)"
+    ldflags="-s -w -X github.com/abiosoft/colima/config.appVersion=${version} \
+    -X github.com/abiosoft/colima/config.revision=$(cat .git-revision)"
```

Comment:
Please set everything static in ldflags attribute.

Reply:
Not possible because the git revision needs to be read from a file during the build.

Reply:
I prefer static where possible and the extra stuff in prebuild but thats just me 🤷

Reply:
Problem is that we'd need to modify the statically declared env variable in preBuild which I think is even less clean than just setting it in preBuild to begin with.

Reply:
@SuperSandro2000 @06kellyjac @Atemu
How shall we proceed here? I am ok with both approaches:
- static ld flags + changing it dynamically in prebuild
- all in configurePhase like right now

Reply:
It doesnt matter too much, whichever you'd prefer since you'll be maintaining it :)

Reply:
Ok, then I'll leave the ldflags as is in the configurePhase.

Reply:
> whichever you'd prefer since you'll be maintaining it :)

well yes but that also does not mean you can do whatever.